src/app/modules/shared/components/import-accounts-form/import-accounts-form.component.ts

```diff
@@ -27,10 +75,19 @@ export class ImportAccountsFormComponent {
       .pipe(
         take(1),
         tap((blob: JSZip) => {
-          blob.forEach(async (item) => {
-            const res = await blob.file(item)?.async('string');
-            if (res) {
-              this.updateImportedKeystores(item, JSON.parse(res));
+          blob.forEach((item) => {
+            const file = blob.file(item);
+            if(file){
+              file.async('string').then((txt) => {
+                try {
+                this.updateImportedKeystores(item, JSON.parse(txt));
+                } catch(err){
+                  // sometimes zip contains extra info that processes like a file
```

Comment:
Any log at least? What if there's an actual error

Reply:
user experience wise it should just look like it didn't receive the files and it won't show up. I was thinking of logging but there would be a clutter or false positives I think...